tests/test_options.py

```diff
@@ -26,3 +28,11 @@ def test_module_in_place(testdir):
     result.assert_outcomes(passed=1)
 
     assert os.path.exists(os.path.join(path, "testfile"))
+
+
+def test_not_existing_venv_option(testdir):
+    testdir.copy_example("testmodule")
+
+    result = testdir.runpytest("tests/test_resource_run.py", "--venv", os.path.join(tempfile.gettempdir(), str(uuid.uuid4())))
+
+    result.assert_outcomes(error=1)
```

Comment:
With or without this patch, this test case will succeed. The goal of this ticket is to improve the error reporting, so it would be better to create an assertion on the exception message.